app/Models/Category.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Category extends Model
+{
+    use SoftDeletes;
+
+    protected $fillable = ['name','slug'];
```

Comment:
```suggestion
    protected $fillable = [
        'name',
        'slug',
    ];
```
tương tự file còn lại nhé